i18n/locales/source/dashboard/base.yml

```diff
@@ -357,7 +357,7 @@ en:
       finished_hoc: "I finished!"
       sign_in_to_save: 'Sign in to save progress'
       my_dashboard: "My Dashboard"
-      course_catalog: "Course Catalog"
+      course_catalog: "Curriculum Catalog"
```

Comment:
you shouldn't need to change anything in the i18n/ directories. They're automatically generated

Reply:
If there wasn't a time crunch here, I'd ask you to revert these files before merging. However, I don't think there's any harm in merging these changes, so it's probably fine to merge with these changes. Tagging @code-dot-org/platform team in case there is something I'm missing here

Reply:
Whoops, my bad. Thanks for the info!